addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -73,6 +73,9 @@ function factory(dependencies) {
             if (this.threadView.channelInvitationForm.component) {
```

Comment:
I guess remove this, double check usage of the "component" field and replace them by the "open state" (future popover relation field) when it is used to check if it is already open